.meta.toml

```diff
@@ -39,11 +53,13 @@ testenv-commands-pre = [
     "{envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} buildout:root-directory={toxinidir}",
     ]
 testenv-commands = [
+    "# The universal2 environment only runs Zope's own tests",
+    "universal2: {envdir}/bin/test {posargs:-vc}",
+    "# All other environments run the expanded alltests script",
     "# the `layer` argument below suppresses a `ZCatalog` performance test",
     "# which occasionally fails on GITHUB",
-    "{envdir}/bin/alltests --layer '!Products.PluginIndexes' {posargs:-vc}",
+    "!universal2: {envdir}/bin/alltests --layer '!Products.PluginIndexes' {posargs:-vc}",
```

Comment:
Did you not want to avoid `alltests` for `universal2`?

By the way: we see above the use of a negative filter (indicated by the leading `!`), applied here to `layer`. Negative filtering is possible for the module, too, and can (among others) be used to exclude tests from dedicated packages (should we want to run `alltests` for `universal2` as well).

Reply:
The negative filter means the alltests script is not executed for the universal2 environment. I don't understand the question.

Reply:
While I get frequently invited to review GHA test related modifications, I am no expert: I have not recognized the `!` as a negative filter.

Reply:
This is actually a `tox` feature. When I say e.g. `tox -e py310-universal2` tox will run the Python 3.10 tests and what's after the dash in the environment name will be directly visible to the tox configuration and I can filter by this name. This filter can be used in many places, like adding/removing step dependencies or commands.